package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rebilly-js-sdk",
-  "version": "41.0.0",
+  "version": "41.0.1",
```

Comment:
That requires increasing major version because you are removing resource.

Reply:
Changed, thanks!